code-challenges.js

```diff
@@ -20,42 +20,107 @@ const fruit4 = "kiwi"
 // --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
 // Pseudo code:
+//  function determines if number below, at, or above boiling point.
+//  paramater: determine where the 3 numbers belong in. 
+//  conditional: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
+ 
+
+//const temp1 = 42
+//const temp2 = 350
+//const temp3 = 212
+
+// const nums = (42) => 
+//   for(let i = 0; i < nums; i ++) {
+//     if(nums[i] % 2 == 0)
+//     console.log(`${nums[i]} below boiling point`)  } 
+ 
+//  const nums2 = [212]
+//     for(let i = 0; i < nums2;  i ++) {
+//       if(nums [i] % 2 ==0)
+//       console.log(`${nums2[i]} at boiling point`)}
+
+//   const num3 = [350]
+// for(let i = 0; i < num3; i ++) {
+//   if (nums [i] % 2 ==0)
+//   console.log(`${num3[i]} above boiling point`)
+//}
+
+//second try for this problem
+
+// const temp = (num) =>
+// if(nums > 212) {
+//  return " is above boiling point"  }
+
+// console.log(temp(350))
 
-const temp1 = 42
-const temp2 = 350
-const temp3 = 212
 
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
 
 // Pseudo code:
+//function; code that combines 2 arrays and gives back the length 
+//paramter;the length of the array 
+//conditional;gives back the correct legth 
 
-const myNumbers1 = [3, 7, 0, 36, -9]
-const myNumbers2 = [8, -7, 42, 9, 13]
+var myNumbers1 = [3, 7, 0, 36, -9]
+var myNumbers2 = [8, -7, 42, 9, 13]
+
+
+//console.log(myNumbers1.concat(myNumbers2).length) 
+//output-10
 
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "2202 atleD"
 
 // Pseudo code:
+//function;reverse the letter of the string 
+//paramter;the reversed string
```

Comment:
I would really like to see pseudocode reflect the code that is provided.  For instance, this is not written as a function so there is not parameter, however, you are using several built in methods that I would like to see here in your pseudocode.  Your code itself looks excellent here though!

Reply:
okay Ill def work on my pseudocode !!